### ISSUE TITLE
allow vectorized input in `r2rtf::rtf_title`, `r2rtf::rtf_footnote` etc 

### ISSUE THREAD

**Issue by elong0527** [Merck/r2rtf#98]
The argument in r2rtf::rtf_title should allow vectorized argument. For example:

```
rtf_title(    
  title = c(       "Sponsor name",       "Table xx.yy.zz"     ),     
  text_justification = c("l", "c")   )
```

**Comment by elong0527** [comment]
Below is an ad-hoc solution, but will need to improve it after addressing #88

```
library(r2rtf)

# Hard coding using RTF syntax
head(iris) %>% 
  rtf_title(title = c("{\\pard\\ql Sponsor name\\par}", "Table xx.yy.zz"),     
            text_justification = c("c")   ) %>% 
  rtf_body() %>%
  rtf_encode() %>%
  write_rtf("tmp1.rtf")

# Customize a text within paragraph.
text <- r2rtf:::rtf_paragraph(r2rtf:::rtf_text("Sponsor name"), justification = "l")
head(iris) %>% 
  rtf_title(title = c(text, "Table xx.yy.zz"),     
            text_justification = c("c")   ) %>% 
  rtf_body() %>%
  rtf_encode() %>%
  write_rtf("tmp2.rtf")
```

**Comment by nanxstats** [comment]
It would also be great if we can cover these cases in the unit tests.

**Comment by panchalh** [comment]
Thanks Yilong, suggested temporary solution of text within paragraph works beautifully to make some titles left justified and center others in the same output.
There is another issue I want to report. In so far how I have used r2rtf, the titles and footnotes created in this package using rtf_title() and rtf_footnote() functions write titles and footnotes in the body of rtf page. By comparison, SAS titles and footnotes go in to the header and footer section of rtf page. Is there a option in r2rtf package to make the titles, footnotes go to the header and footer sections of rtf respectively?

Thank you

![rtf_header_body_footer__data_Sashelp_dot_Class](https://user-images.githubusercontent.com/36645928/167927921-0b2667f2-045c-40f3-8f00-98804a03534c.PNG)

**Comment by elong0527** [comment]
Please refer 

https://merck.github.io/r2rtf/reference/rtf_page_header.html
https://merck.github.io/r2rtf/reference/rtf_page_footer.html

and a summary of function can be found in https://r4csr.org/slides/r4csr-gwu.html#20

**Comment by elong0527** [comment]
Fixed in #100

Example code:

```
head(iris) %>% 
  rtf_title(title = c("abc", "Table xx.yy.zz"),     
            text_justification = c("l", "c")   ) %>% 
  rtf_body() %>%
  rtf_encode() %>%
  write_rtf("tmp1.rtf")
```
